orchestration/src/main/java/com/sap/ai/sdk/orchestration/ContentItem.java

```diff
@@ -0,0 +1,4 @@
+package com.sap.ai.sdk.orchestration;
+
+/** Represents an item in a {@link MessageContent} object. */
+public sealed interface ContentItem permits TextItem, ImageItem {}
```

Comment:
(Minor)

Please make sure to add `@since 1.3.0` on all new public API (new public classes and new public methods on old classes)